experimental/dds/tree/package.json

```diff
@@ -22,16 +22,6 @@
 				"types": "./dist/index.d.ts",
 				"default": "./dist/index.js"
 			}
-		},
-		"./test/EditLog": {
```

Comment:
The safer change would be to keep this intact until we make breaking changes.
I don't know if anyone is using this export. Loop code base mentions it, but that appears to be metadata.
The safest change would be to create an EditLogDeprecated.ts file that re-exports EditLog.ts and adds `@deprecated` tag to each export. However, I don't think we need that as far as I can tell **everything** is already marked `@alpha` which lets them and then whole export go away at any time.

Reply:
Kept as removed, as everything is marked as `@alpha` as mentioned